### PR TITLE
Update connectivity-architecture.md

### PR DESCRIPTION
**Pull request by toastjoqo**
I would suggest not using “Azure synapse” for formerly SQL DW, because they are different:

Servers in SQL Database and formerly SQL DW support the following three options for the server's connection policy setting:
Note: The connection policy for Synapse SQL in Azure Synapse Analytics is set to Default. You cannot change this in Azure Synapse Analytics.

refer to synapse workspace connectivity setting: https://docs.microsoft.com/en-us/azure/synapse-analytics/security/connectivity-settings#connection-policy

### CODE REVIEW

**Comment by PRMerger14** [comment]
@toastjoqo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Jak-MS** [comment]
@rohitnayakmsft 

Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by rohitnayakmsft** [comment]
@Jak-MS  the change looks fine by me. Please have someone from the Synapse team review it as well to make sure the wording is consistent with their docs

File: azure-sql/database/connectivity-architecture.md [hunk 1]
```diff
@@ -36,7 +36,8 @@ The following steps describe how a connection is established to Azure SQL Databa
 
 ## Connection policy
 
-Servers in SQL Database and Azure Synapse support the following three options for the server's connection policy setting:
+Servers in SQL Database and formerly SQL DW support the following three options for the server's connection policy setting:
+Note: The connection policy for Synapse SQL in Azure Synapse Analytics is set to Default. You cannot change this in Azure Synapse Analytics. Refer to [synapse workspace connectivity setting](https://docs.microsoft.com/en-us/azure/synapse-analytics/security/connectivity-settings#connection-policy).
```
Comment (MashaMSFT) [inline azure-sql/database/connectivity-architecture.md:40]:
```suggestion

> [!NOTE]
> The connection policy for Synapse SQL in Azure Synapse Analytics is set to Default. You cannot change this in Azure Synapse Analytics. Refer to [synapse workspace connectivity setting](/azure/synapse-analytics/security/connectivity-settings#connection-policy).
```

**Comment by MashaMSFT** [comment]
Hi @toastjoqo , thanks for helping improve our docs! Before I validate this change with the Azure Synapse team, would you mind committing my suggestion? 

Thanks!
Masha from the SQL Docs team

**Comment by WilliamDAssafMSFT** [comment]
Thanks @toastjoqo for this good suggestion. This needs slightly different wording: The connection policy for dedicated SQL pools (formerly SQL DW) in Azure Synapse Analytics is set to **Default**. You cannot change this for dedicated SQL pools in Synapse workspaces. Will address in https://github.com/MicrosoftDocs/sql-docs-pr/pull/24248/

**Pull request closed without merging**